docs/deployment/README.md

```diff
@@ -333,6 +333,20 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
 ----
 ## Deploy Cumulus dashboard
 
+### Dashboard Requirements
+
+Please note that the requirements are similar to the [Cumulus stack deployment requirements](#Requirements), however the node version may vary slightly and the dashboard requires yarn.    The installation instructions below include a step that will install/use the required node version referenced in the `.nvmrc` file in the dashboard repository.
+
+- git
+- [node 8.11.4](https://nodejs.org/en/) (use [nvm](https://github.com/creationix/nvm) to upgrade/downgrade)
+- [npm](https://www.npmjs.com/get-npm)
+- [yarn](https://yarnpkg.com/en/docs/install#mac-stable)
+- sha1sum or md5sha1sum
+- zip
+
+- AWS CLI - [AWS command line interface](https://aws.amazon.com/cli/)
+- python
```

Comment:
I know this isn't you, but on line 354 the 'preparing aws configuration' link doesn't work either.

Reply:
Hmm, that's  neat.  I'll see about fixing that.

Reply:
46df58a

Reply:
It wasn't just that one either, the doc had several instances of that kind of link.  I fixed the deployment doc instances in the above commit.

Reply:
👍 thanks!